pkgs/development/libraries/vtk/9.x.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, fetchurl, cmake, libGLU, libGL, libX11, xorgproto, libXt, libtiff
+, fetchpatch
+, qtLib ? null
```

Comment:
is this suppose to be QtLibs? please take a look at https://nixos.org/nixpkgs/manual/#sec-language-qt , doesn't follow normal qt conventions

Reply:
I think it's fixed now